daemon/config.go

```diff
@@ -51,39 +51,131 @@ type Config struct {
 	LogConfig                   runconfig.LogConfig
 }
 
+func addDaemonStringVar(p *string, names []string, value string, usage string) {
+	flag.StringVar(p, names, value, usage)
+	for _, opt := range names {
+		if opt[:1] == "#" {
+			opt = opt[1:]
+		}
```

Comment:
Seems like a good use of [`string.TrimPrefix`](http://golang.org/pkg/strings/#TrimPrefix):
    opt = strings.TrimPrefix(opt, "#")